packages/mask/src/components/shared/ApplicationBoard.tsx

```diff
@@ -301,7 +309,15 @@ export function ApplicationBoard({ secondEntries, secondEntryChainTabs }: MaskAp
                             () => {},
                         ),
                     ],
-                    [ChainId.Mainnet, ChainId.BSC, ChainId.Matic, ChainId.Arbitrum, ChainId.xDai, ChainId.Celo],
+                    [
```

Comment:
may be can extract as multi-presets.

Reply:
They are used only in this file, not very common, and extracting will decrease the flexibility. We can do it anytime when we need to.